Refit/ValidationApiException.cs

```diff
@@ -27,7 +27,6 @@ public static ValidationApiException Create(ApiException exception)
         /// <summary>
         /// The problem details of the RFC 7807 validation exception.
         /// </summary>
-        public new ProblemDetails Content => GetContentAs<ProblemDetails>();
-
+        public new ProblemDetails Content => GetContentAsAsync<ProblemDetails>().ConfigureAwait(false).GetAwaiter().GetResult();
```

Comment:
This can deadlock. Async calls need to be awaited, thus cannot really be a property.